aws/dms/tasks.yml

```diff
@@ -68,10 +68,6 @@ cron-pii:
 # All tables in this section are exported to the Redshift `dashboard_production` schema. Do NOT export columns containing
 # sensitive data to this schema.
 cron:
-- dashboard.ai_tutor_interactions:
```

Comment:
In addition to moving this to `cron-pii`, looks like we're inadvertently removing the table entirely from our MySQL --> Redshift export. We only wan to delete the `remove_column` key and it's nested array.

Reply:
Looking back at what we [requested for `aichat` tables](https://github.com/code-dot-org/code-dot-org/pull/62257/files), after the migration we worked with Allison on creating a [saved query](https://app.trevor.io/datasources/9b7ea657-edf4-44a1-a24b-f86147b64075/saved-queries/86534762-8685-41d3-af69-8a4ac1057b01) that product can run that they can then build their dashboards on. 
Sorry about not mentioning this earlier!

Reply:
I'm still confused. `ai_tutor_interactions` is _already_ in `cron_pii` on line 60. 🤔

Reply:
I believe you can currently write a query accessing all the columns in `ai_tutor_interactions` (including `prompt` and `response`) under 'Raw Tables'.

Reply:
⚡ thank you, Alice! I didn't realize there was already a way to access tables exported via cron_pii in Trevor 🎉